### PR TITLE
Incorporate optional user hijack

### PR DESCRIPTION
**Pull request by jrbotros**
Adds optional [django-hijack](https://github.com/arteria/django-hijack/blob/master/README.md) functionality. This can be added to a project by specifying the correct URLs (e.g., `url(r'^orchestra/switch/', include('hijack.urls'))`) and by including `'django.core.context_processors.request'` in your list of template context_processors.

### CODE REVIEW

File: example_project/example_project/urls.py [hunk 1]
```diff
@@ -21,6 +21,9 @@
     url(r'^orchestra/accounts/',
         include('registration.backends.default.urls')),
 
+    # Optionally include these routes to enable user hijack functionality.
+    url(r'^orchestra/switch/', include('hijack.urls')),
```
Comment (thisisdhaas) [inline example_project/example_project/urls.py:25]: please update docs, as this changes [install instructions](http://orchestra.readthedocs.org/en/stable/getting_started.html#install-and-configure-orchestra)

File: example_project/example_project/settings.py [hunk 2]
```diff
@@ -63,9 +63,11 @@
         'OPTIONS': {
             'context_processors': [
                 'django.template.context_processors.debug',
-                'django.template.context_processors.request',
                 'django.contrib.auth.context_processors.auth',
                 'django.contrib.messages.context_processors.messages',
+                # Allows access to request object in templates; currently used
+                # for displaying hijack notification
+                'django.template.context_processors.request',
```
Comment (thisisdhaas) [inline example_project/example_project/settings.py:68]: Why did this get moved?

File: example_project/example_project/urls.py (hunk 1 above)
Comment (jrbotros) [inline example_project/example_project/urls.py:25]: :white_check_mark:

File: example_project/example_project/settings.py (hunk 2 above)
Comment (jrbotros) [inline example_project/example_project/settings.py:68]: No real reason, I'll move it back by the other `.template` one.

Comment (jrbotros) [inline example_project/example_project/settings.py:68]: :white_check_mark:

**Comment by thisisdhaas** [comment]
LGTM

land it
LAND IT.

**Pull request merged**